src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,51 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { MovieData } from '../../types/MovieData';
+
+type Props = {
+  addMovie: (newMovie: any) => void
```

Comment:
Avoid using any. Specify the type
![image](https://user-images.githubusercontent.com/96409563/204758210-2bfacbf8-d5b6-4b2e-96a9-f3113aaad2f9.png)